### ISSUE TITLE
No content displayed until one option is selected

### ISSUE THREAD

**Issue by acelaya** [jthegedus/docsify-select#7]
**Describe the bug**

I'm using a select to show documentation related with different versions for a library.

When the page loads for the first time, it only displays the select, with the first option selected, but no content is displayed.

If I change to some other option, then it starts displaying the content for that option.

After that, if I go back to the first option, it also displays its content properly. 

**Steps to reproduce**

I have created this document that can be used to test:

```md
<!-- select:start -->
<!-- select-menu-labels: Version -->

### --v8.x--

Content for v8...

### --v7.x--

Content for v7...

<!-- select:end -->
```

**Expected behavior**

When the page loads, it shows the first version selected, together with its content.

In the example above, it should display v8.x selected, and the content should be `Content for v8...`.

**Screenshots**

When the page loads, it shows this:

![image](https://user-images.githubusercontent.com/2719332/88048440-a047b700-cb53-11ea-93f4-53dd7e9a6a2d.png)

Then If I select v7.x from the select, it shows this:

![image](https://user-images.githubusercontent.com/2719332/88048484-b48bb400-cb53-11ea-9d6c-72ea6c2bc499.png)

If I go back to v8.x, then this is displayed:

![image](https://user-images.githubusercontent.com/2719332/88048557-d2591900-cb53-11ea-9b55-12b7b94824d6.png)

**Additional context**

*N/A*

**Comment by acelaya** [comment]
It looks like the bug is inside this function: https://github.com/jthegedus/docsify-select/blob/master/src/index.js#L271

All the code there is wrapped inside the `if (settings.detectOperatingSystem.enabled) {...}`, which was disabled for me, as I don't need to detect the OS.

Once I enabled it, it started to render the first option properly.

I think moving this part out of the `if`, should solve the issue:

```js
// Now do the same as changeSelection function
const newSelection = calculateSelectedContent(selectMenuList);
const contentMatch = setSelectedContent(newSelection, selectContentList);
// If at this point no element has the 'data-select-content' value, then set the default if it exists
if (contentMatch === false) {
	setDefaultContent(selectContentList);
}
```

But I haven't checked. It's just a blind guess. If you cpnfirm this is a bug and not the desired behavior, I can investigate a big more and provide the PR to fix it.

**Comment by jthegedus** [comment]
Sorry, for some reason I did not get notifications of Issues in this repo. I will investigate.

**Comment by acelaya** [comment]
No problem 🙂 

In case it helps reproducing it, these are the docs in which I'm using docsify-select https://github.com/acmailer/acmailer/tree/main/docs

You can see it in action here https://acmailer.github.io/acmailer/#/installation

For the moment, I worked around the issue by enabling the OS detection https://github.com/acmailer/acmailer/blob/main/docs/index.html#L24

**Comment by jthegedus** [comment]
That information will help a lot, thanks :+1:

**Comment by acelaya** [comment]
I'm a bit confused here. You have just released v0.3.0, but there was already a v1.0.0.

Is this v0.3 supposed to have everything that was already in v1.0.0? And if the answer is yes, why not v1.1.0 instead?

**Comment by jthegedus** [comment]
Hey @acelaya 

I moved to using [semantic-release](https://github.com/semantic-release/semantic-release) and it calculates the version to publish based off of git commit history and the current, since I didn't signal a breaking change anywhere in the commit history it erroneously detected `0.3.0` as the next release number.

I am yet to rectify this and publish these changes under `1.1.0`, should be in the next couple of days. Apologies for the confusion.

**Comment by acelaya** [comment]
All clear. I guessed the reason was somehow related with automation.

Thanks for the clarification 🙂

**Comment by jthegedus** [comment]
Sometimes doing something manually is the way to go.

**Comment by jthegedus** [comment]
I resolved the package # issue. You should see this fix in `v1.1.0`
